### PR TITLE
[G&M] Fully Show Group Creation Options

### PR DESCRIPTION
**Pull request by smmr-dn**
Fixed bug where it prevents group creation options to be fully shown when there is space.
image

### CODE REVIEW

File: packages/itwin/grouping-mapping-widget/src/widget/components/customUI/DefaultGroupingUI.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const defaultGroupingUI: GroupingMappingCustomUI[] = [
   },
   {
     name: "Manual",
-    displayLabel: "Manual",
+    displayLabel: "Manual Query",
```
Comment (Guillar1) [inline packages/itwin/grouping-mapping-widget/src/widget/components/customUI/DefaultGroupingUI.tsx:30]: Why was this renamed?

Comment (smmr-dn) [reply packages/itwin/grouping-mapping-widget/src/widget/components/customUI/DefaultGroupingUI.tsx:30]: I saw in both the backlog item image for this bug (there was a "..." following "Manual") and the readme file under the grouping-mapping-widget folder where it was specified as "Manual Query". I think it is clearer too now that we have space, but I should have asked first.

Comment (Guillar1) [reply packages/itwin/grouping-mapping-widget/src/widget/components/customUI/DefaultGroupingUI.tsx:30]: No worries. I think I renamed it to Manuel Query when I did a whole refactor of this widget then it was caught my QA asking the same question, I believe they took that screenshot when I made that mistake.

Comment (Guillar1) [reply packages/itwin/grouping-mapping-widget/src/widget/components/customUI/DefaultGroupingUI.tsx:30]: Yes

**Review by Guillar1 — APPROVED** [review]
